android/app/build.gradle

```diff
@@ -138,7 +138,7 @@ def secrets = loadProjectSecrets(project.env.get("DEFAULT_TESTNET"))
 
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
-    namespace "org.celo.mobile"
+    namespace "org.mobilestack.mobile"
```

Comment:
should this be `xyz.mobilestack.app`, where we would update `app` to the app name?